index.html

```diff
@@ -10,10 +10,54 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
-
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
-
+ <header>
+    <a class="logo" href="index.html"><img src="img/karma-logo.svg" alt="karma logo"></a>
+    <nav>
+        <ul class="list">
+            <li class="items"><a class="links" href="#"> Meet Karma</a></li>
+            <li class="items"><a class="links" href="#"> How it Works</a></li>
+            <li class="items"><a class="links" href="#"> Store</a></li>
+            <li class="items"><a class="links" href="#"> Blog</a></li>
+            <li class="items"><a class="links" href="#"> Help</a></li>
+            <li class="items"><a class="links" href="#"> Login</a></li>  
+        </ul>     
+    </nav>
+ </header> 
+ <main> 
+    <section class="p1">
```

Comment:
Really good clean markup structure, well done. You've used header, main, footer at the top level. `<section />` is a good choice for the containers below that (rather than article, because that's for content that could be shared / syndicated with other sites).